service/src/main/java/uk/gov/hmcts/reform/fpl/service/ApplicantService.java

```diff
@@ -3,37 +3,73 @@
 import com.google.common.collect.ImmutableList;
 import org.springframework.stereotype.Service;
 import uk.gov.hmcts.reform.fpl.enums.PartyType;
+import uk.gov.hmcts.reform.fpl.model.Address;
 import uk.gov.hmcts.reform.fpl.model.Applicant;
 import uk.gov.hmcts.reform.fpl.model.ApplicantParty;
 import uk.gov.hmcts.reform.fpl.model.CaseData;
 import uk.gov.hmcts.reform.fpl.model.common.Element;
+import uk.gov.hmcts.reform.rd.model.ContactInformation;
+import uk.gov.hmcts.reform.rd.model.Organisation;
 
 import java.util.ArrayList;
 import java.util.List;
 import java.util.UUID;
 
+import static java.util.Objects.isNull;
 import static java.util.stream.Collectors.toList;
 import static org.springframework.util.ObjectUtils.isEmpty;
 
 @Service
 public class ApplicantService {
 
-    public List<Element<Applicant>> expandApplicantCollection(CaseData caseData) {
+    public List<Element<Applicant>> expandApplicantCollection(CaseData caseData, Organisation organisation) {
         if (isEmpty(caseData.getApplicants())) {
-            return ImmutableList.of(Element.<Applicant>builder()
-                .value(Applicant.builder()
-                    .party(ApplicantParty.builder()
-                        // A value within applicant party needs to be set in order to expand UI view.
-                        .partyId(UUID.randomUUID().toString())
+            if (isEmpty(organisation)) {
```

Comment:
is this going to catch what you expect? We are always passing an organisation here due to the builder

Reply:
could we actually be clever here and always add organisation to applicant, wrap the values of organisation in optionals sort of thing? Removes the need for buildApplicantWithOrganisationDetails method, and the need for this isEmpty method

Reply:
I've added the organisation always here, it isn't using optionals but have a look and see what you think. :) i've removed need for buildApplicantWithOrganisationDetails method